controllers/kafkamanagement/kafkauser_controller.go

```diff
@@ -414,26 +342,15 @@ func (r *KafkaUserReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	if user.DeletionTimestamp != nil {
 		for clusterID, clusterEvent := range user.Status.ClustersEvents {
 			if clusterEvent == models.Created || clusterEvent == models.CreatingEvent || clusterEvent == models.UpdatingEvent || clusterEvent == models.UpdatedEvent {
-				l.Error(models.ErrUserStillExist, "please remove the user from the cluster specification",
+				l.Error(models.ErrUserStillExist, "If you want to delete the user, first remove the user from cluster specification",
 					"username", username, "cluster ID", clusterID)
 				r.EventRecorder.Event(user, models.Warning, models.DeletingEvent,
-					"The user is still attached to cluster, please remove the user from the cluster specification.")
+					"If you want to delete the user, first remove the user from cluster specification")
```

Comment:
Please use here `msgDeleteUser` as you do in other similar cases

Reply:
This const is in a different package. I'll think about what to do.